systemtests/tests/config-syntax-crash/testrunner

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Run a simple backup
```

Comment:
A comment could help to understand that the config does not crash the director because of the invalid ACL Syntax.

Reply:
The unittest looks reasonable.